src/conversion_helper.js

```diff
@@ -1,20 +1,22 @@
 const rh = require('./regex_helper');
+let inRangeVal = -Infinity;
 
 function distanceMap(m) {
```

Comment:
Maybe one solution to get rid of "inRangeVal" is for these functions to accept an array instead of just a single number

Reply:
Hmm, I'll do that. Sounds much neater.